syft/core/frameworks/torch/tensor.py

```diff
@@ -1559,95 +1559,88 @@ def __gt__(self, other):
     def __le__(self, other):
 
         a, b = self.check_and_scale_precision_if_needed(other)
         result = (a <= b).long() * self.base ** self.precision_fractional
-        result = sy._FixedPrecisionTensor(
-            result,
-            base=self.base,
-            field=self.field,
-            precision_fractional=self.precision_fractional,
-            precision_integral=self.precision_integral,
-            already_encoded=True,
-        ).wrap(True)
+
+        result = sy._FixedPrecisionTensor(result,
+                                          base=self.base,
+                                          field=self.field,
+                                          precision_fractional=self.precision_fractional,
+                                          precision_integral=self.precision_integral,
+                                          already_encoded=True).wrap(True)
+
         return result
 
     def __eq__(self, other):
 
         a, b = self.check_and_scale_precision_if_needed(other)
         result = (a == b).long() * self.base ** self.precision_fractional
-        result = sy._FixedPrecisionTensor(
-            result,
-            base=self.base,
-            field=self.field,
-            precision_fractional=self.precision_fractional,
-            precision_integral=self.precision_integral,
-            already_encoded=True,
-        ).wrap(True)
+
+        result = sy._FixedPrecisionTensor(result,
+                                          base=self.base,
+                                          field=self.field,
+                                          precision_fractional=self.precision_fractional,
+                                          precision_integral=self.precision_integral,
+                                          already_encoded=True).wrap(True)
+
         return result
 
     def __div__(self, other):
         # if other is not a fixed tensor, convert it to a fixed one
-        if not hasattr(other, "precision_fractional"):
-            other = other.fix_precision(precision_fractional=self.precision_fractional)
 
-        if self.precision_fractional == other.precision_fractional:
-            gp_response = (
-                self.child * 10 ** self.precision_fractional / other.child
-            ) % self.field
-        elif self.precision_fractional > other.precision_fractional:
-            gp_response = (
-                self.child / other.child * 10 ** other.precision_fractional
-            ) % self.field
-
-        elif self.precision_fractional < other.precision_fractional:
-            gp_response = (
-                (
-                    self.child
-                    * 10 ** (2 * other.precision_fractional - self.precision_fractional)
-                )
-                / other.child
-            ) % self.field
-        return gp_response
+        if (not hasattr(other, 'precision_fractional')):
+            other = other.fix_precision(precision_fractional = self.precision_fractional)
```

Comment:
It's just an indentation error, fix the indentations on lines `1632`, `1635`, `1637` and `1640`. That should fix the build error.